addons/project/models/project.py

```diff
@@ -2760,6 +2760,8 @@ def arrange_tag_list_by_id(self, tag_list, id_order):
     def _name_search(self, name='', args=None, operator='ilike', limit=100, name_get_uid=None):
         ids = []
         if not (name == '' and operator in ('like', 'ilike')):
+            if not args:
```

Comment:
```suggestion
            if args is None:
```
this way, we don't reassign args if it is equal to [] already.